### DISCUSSION TITLE
Restoring only the last 5 tabs

### DISCUSSION THREAD

**Discussion by spkoder** [sumatrapdfreader/sumatrapdf#4596] [Ideas]
Hello, please add a setting that would specify how many tabs to restore when starting the program, for example, 5.
The fact is that when a lot of tabs are restored, the program takes a long time to run.
It would also be nice to add a setting that would set the number of recent documents whose settings would be stored in the program

**Comment by ramiropistoia** [comment]
I like this. And also put in the settings menu the options NoHomeTab and RestoreSession of the INI file, for ease

**Comment by Birador** [comment]
It get really annoying to  wait for your 10kb file to open when a 10MB file is being prefetched, just show me the tabs but dont open the files till i click to open the tab to open it. Else fetch them after opening the requested file. Also If i want to close multiple tabs with my mouse the tab size keep changing and close tab button keeps running away... minor issue.
